src/pages/resources/index.js

```diff
@@ -42,7 +43,7 @@ const MembersPage = () => {
         <GlobalStyle />
         <SEO title="Service Mesh Resources" description="Articles on how to service mesh from the world's largest service mesh community. Service mesh how-tos and cloud native ecosystem news." canonical="https://layer5.io/resources" />
         <Navigation />
-        <PageHeader title="Cloud Native Resources" path="Resources" subtitle="Learn how to provision, secure, connect, and run any Kubernetes and any service mesh for any application." />
+        <PageHeader title="Cloud Native Resources" path="Resources" subtitle="Learn how to provision, secure, connect, and run any Kubernetes and any service mesh for any application." img={RssFeedIcon} feedlink="/blog/feed.xml"/>
```

Comment:
@Sunnyio change feedlink here to : "/resources/feed.xml"

Reply:
my bad, changing it now.

Reply:
@Abhijay007  can you review it now?